content/chapter-04/NetworkManagementLab.md

```diff
@@ -1,11 +1,61 @@
 # Lab: Cloud Network Management
 
-## Objectives
-
 * Get familiar with Cloud Provider access control management
-* Review options available for VPN connectivity on the portal
+* You will create three small instances on this lab:
+  * **In-Access**: Accessible from outside your network on port 80
+  * **Middle-Access** Accessible only from **In-Access** and **Only-internal** on port 80
+  * **Only-internal** Accessible from **In-Access** and **Middle-Access**, but without access to the outside world
+* Login to Cloud Provider portal
+* Create a security group to allow access to SSH port 22 and the outside world. Let's call this security group `chapter4`
```

Comment:
Create a security group to allow inbound access on SSH port 22 from the outside world

Let's call this security group `chapter4` - how multiple students will be able to create a security group with the same name?

Reply:
Good point! Changing it to <student-id>-chapter4 and the instructor provide the id or the student choose one.